contracts/mocks/HoldableMock.sol

```diff
@@ -7,6 +7,8 @@ contract HoldableMock is Holdable {
     bool isExpiredSet;
     bool isExpired;
 
+    event RevokedHoldOperator(address indexed operator, address indexed account);
```

Comment:
The event shoulfd be HoldOperatorRevoked as also need to change also HoldOperatorAuthotized. Its legacy but i think it must be changed

Reply:
Thanks for the correction. In fact, It is not necessary and I will remove it.

By another hand, I will change `HoldOperatorAuthorized` to solve the legacy mistake.